pkg/messaging/rabbitmq/pubsub_test.go

```diff
@@ -0,0 +1,243 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package rabbitmq_test
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/mainflux/mainflux/pkg/messaging"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	topic       = "topic"
+	chansPrefix = "channels"
+	channel     = "9b7b1b3f-b1b0-46a8-a717-b8213f9eda3b"
+	subtopic    = "engine"
+)
+
+var (
+	msgChan = make(chan messaging.Message)
+	data    = []byte("payload")
+)
+
+func TestPubsub(t *testing.T) {
+	pubsubcases := []struct {
```

Comment:
Fix typos: `sub topic` -> `subtopic` and `Unsusbcribe` -> `Unsubscribe`.

Reply:
Done